types/artifacts.d.ts

```diff
@@ -213,6 +215,36 @@ declare global {
         requestId: string | null
       }
 
+      /** @see https://sourcemaps.info/spec.html#h.qz3o9nc69um5 */
+      export type RawSourceMap = {
+        /** File version (always the first entry in the object) and must be a positive integer. */
+        version: number
+        /** A list of original sources used by the “mappings” entry. */
+        sources: string[]
+        /** A list of symbol names used by the “mappings” entry. */
+        names: string[]
+        /** An optional source root, useful for relocating source files on a server or removing repeated values in the “sources” entry. This value is prepended to the individual entries in the “source” field. */
+        sourceRoot?: string
+        /** An optional list of source content, useful when the “source” can’t be hosted. The contents are listed in the same order as the sources in line 5. “null” may be used if some original sources should be retrieved by name. */
```

Comment:
> “null” may be used if some original sources should be retrieved by name.

I don't think this is relevant for audits? (I think they're going to have match up to `ScriptElements`)

Reply:
hm.. apparently this means if a contents is `null`, you should be able to load the source map via URL from the server? if so, yeah this doesn't matter at the artifact level. perhaps we could load source contents via the server this way (if we see nulls here), but that's kinda weird. wonder what DevTools does.